tests/prompt_strategies/messages/test_chat.py

```
@@ -4,13 +1,23 @@
+"""Module for testing chat message internals."""
 
-# pylint: disable=duplicate-code
-import logging
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)